docs/ref-info/ref-tokens.md

```diff
@@ -49,6 +51,36 @@ The recommended refresh interval is hourly.
 
 To determine when to refresh, you can use the timestamp of the `refresh_from` field in the response to the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint (see [Successful Response](../endpoints/post-token-generate.md#successful-response)) or [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoint (see [Successful Response With Tokens](../endpoints/post-token-refresh.md#successful-response-with-tokens)). The value of this field is a timestamp in UNIX time, expressed in milliseconds.
 
+### Managing Token Refresh with an SDK
+
+An easy way to manage token refresh is to use one of the UID2 SDKs that have a function for the purpose: either the Java or Python SDK.
+
+Each of these SDKs includes a class, `Uid2PublisherClient`, that has a function to determine if a token refresh is needed.
+
+The following examples show how you could check if refresh is needed using one of these functions. The function checks whether you should refresh, and then, if needed, you can easily call the refresh function to refresh the token.
```

Comment:
I suggest to add the section "Determine if the identity can be refreshed (that is, the refresh token hasn't expired)" also.  
<img width="978" alt="Screenshot 2024-09-19 at 12 30 46 PM" src="https://github.com/user-attachments/assets/f32b934f-1866-43a4-b51a-fb4104049fba">

Reply:
Done.